infra/repo_manager.py

```diff
@@ -146,6 +146,35 @@ def get_commit_list(self, old_commit, new_commit):
     commits.append(old_commit)
     return commits
 
+  def _get_git_history(self):
+    """Gets the current git repository history."""
+    git_path = os.path.join(self.repo_dir, '.git', 'shallow')
+    if os.path.exists(git_path):
+      build_specified_commit.execute(['git', 'fetch', '--unshallow'],
```

Comment:
`execute` should probably be somewhere other than `build_specific_commit`

Reply:
I am going to push that here: https://github.com/google/oss-fuzz/pull/3313